core/data-transformers/form-field-transformer/index.ts

```diff
@@ -56,7 +62,18 @@ export const formFieldTransformer = (
       };
 
     case 'PicklistFormField':
+      if (field.entityId === FieldNameToFieldId.countryCode) {
+        return {
+          id: String(field.entityId),
+          type: 'text',
+          name: String(field.entityId),
+          label: field.label,
+          required: field.isRequired,
+        };
+      }
```

Comment:
In a follow up PR, I think we need to use our `Geography` node to populate a dropdown here instead of a text field. The API requires a country code (`US`), but autofill will copy over the entire string (`United States`).

Reply:
Added in https://github.com/bigcommerce/catalyst/pull/1878